trunk/cli/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pg-trunk"
-version = "0.3.2"
+version = "0.4.0-alpha.1"
```

Comment:
we have to republish packages since they are in a new format, so I don't want it as the default on cargo install for now